tests/aws/scenario/bookstore/test_bookstore.snapshot.json

```diff
@@ -906,7 +906,7 @@
           {
             "SourceVersion": "OpenSearch_2.5",
             "TargetVersions": [
-              "OpenSearch_2.7", "OpenSearch_2.9"
+              "OpenSearch_2.7", "OpenSearch_2.9", "OpenSearch_2.11"
```

Comment:
It looks a bit like this snapshot was updated manually, which is an anti-pattern (the whole purpose of the snapshot framework is that we know that the responses from LocalStack look the same as the ones from AWS).
Please update the snapshot against AWS and then validate it against LocalStack.
You can do that easily by running the test against your default profile in AWS with these settings:
- `SNAPSHOT_UPDATE=1`
- `TEST_TARGET=AWS_CLOUD`

Reply:
I updated the snapshot in the latest commit.